docs/guide/breakpoint-preview.md

```diff
@@ -108,7 +103,22 @@ Certain media query conditions are not supported or do not translate well to con
     @media (min-aspect-ratio: 16/9) { ... }
     ```
 
-#### 2. **Behavioral Differences**
+#### 2. Operator and Descriptor Limitations
+Some combinations of size-related descriptors and operators may not work reliably:
+
+When `min-width`, `max-width`, `min-height`, or `max-height` are used with `>=` or `<=` operators.
+
+- Example of problematic query:
+  ```css
+    @media (min-width >= 600px) { ... }
```

Comment:
This is accurate documentation, I know.

In terms of the feature... is it just silly? I get it, min-width is already a ">=" situation, it's bizarre to add an explicit ">="...

Reply:
Yeah it is just silly isn't it. I think it would be good for us to explicitly say "adding >= to min-width doesn't make sense because it is already a greater-than-or-equal comparison, so we don't support it."

Reply:
Updated